int32[72] index_tip_Value
int32[36] index_mid_Value
int32[36] index_end_Value

int32[72] middle_tip_Value
int32[36] middle_mid_Value
int32[36] middle_end_Value

int32[72] ring_tip_Value
int32[36] ring_mid_Value
int32[36] ring_end_Value

int32[72] thumb_tip_Value
int32[36] thumb_mid_Value

int32[113] palm_Value
